code/modules/chemistry/Reagents-Misc.dm

```diff
@@ -3069,11 +3069,23 @@ datum
 
 			on_mob_life(var/mob/M, var/mult = 1) // cogwerks note. making atrazine toxic
 				if (!M) M = holder.my_atom
-				M.take_toxin_damage(2 * mult)
+				if (istype(M, /mob/living/critter/plant))
+					M.take_toxin_damage(3 * mult)
+				else
+					M.take_toxin_damage(2 * mult)
 				flush(holder, 2 * mult, flushed_reagents)
 				..()
 				return
 
+			reaction_mob(var/mob/M, var/method=TOUCH, var/volume)
+				var/datum/reagent/self = src					  //of the reagent to the mob on TOUCHING it.
```

Comment:
???

Reply:
I was not sure about this and wondering if it was really needed when i copied it out of the parent. In hindsight i should have folloed the thought and throw it out.